### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by gergana95**
screen shot 2017-07-07 at 10 51 29 am

Added:
- header font
- styling to header

Tasks:
- [x] navigation alignment

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,20 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
+<div class="wrapper">
+	 
+	 <header class="header">
+	 	<img class="header__logo" src="images/logo.svg" alt="header logo">
```
Comment (superKalo) [inline index.html]: Please get familiar with the [different ways of using SVG](https://css-tricks.com/using-svg/).

File: index.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+/* Header styling */
+.wrapper {
+
+}
+
+.header {
+	width: 612px;
+	height: 44px;
+    margin: 38px 720px 0 108px;
```
Comment (superKalo) [inline index.css]: No :)

File: index.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    margin-top: 15px;
+    float: right;
+}
+
+.list__item {
+    list-style-type: none;
+    float: left;
+    height: 18px;
+    margin-right: 43px;
+}
```
Comment (superKalo) [inline index.css]: There is an easier way how to display the list with items on one line:

```
ul {
    margin-left: 0;
    list-style: none;
}
li {
    display: inline-block;
}
```

File: index.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+.header__logo {
+	width: 16px;
+	height: 44px;
+}
+
+.header__nav {
+    display: inline-block;
+    width: 406px;
```
Comment (superKalo) [inline index.css]: Must be dynamic, not a hardcoded value.

File: index.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+.header {
+	width: 612px;
+	height: 44px;
+    margin: 38px 720px 0 108px;
+    
+}
+
+.list {
+    width: 100%;
+    height: 100%;
```
Comment (superKalo) [inline index.css]: `width` and `height` doesn't really do anything here

File: index.css [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        url('fonts/ssubset-LeagueSpartan-Bold.woff') format('woff'),
+        url('fonts/subset-LeagueSpartan-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+/* Header styling */
+.wrapper {
+
+}
```
Comment (superKalo) [inline index.css]: Yes! But why you abandoned it? 🔮

**Comment by superKalo** [comment]
Error in the console:
```
GET http://localhost/surfing/fonts/ssubset-LeagueSpartan-Bold.woff 
```

File: index.css [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    margin:0;
+}
+
+.header__logo {
+	width: 16px;
+	height: 44px;
+}
```
Comment (superKalo) [inline index.css]: The SVG itself has width and height, so probably these won't make a lot of sense here, but they might do in the `` tag. [Here's why](https://stackoverflow.com/a/1247693/1333836).

**Comment by superKalo** [comment]
Double check that you're using the [global conventions](https://github.com/dev-labs-bg/devlabs-internal-code-conventions#Основни-настройки-на-code-editor-а).

File: index.css [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    height: 18px;
+    margin-right: 43px;
+}
+
+.list__item:last-child {
+    margin-right: 0;
+}
+
+.list__link {
+    font-family: "League Spartan";
```
Comment (superKalo) [inline index.css]: Google it first.
```
.font-primary {
    font-family: "League Spartan", Arial, Tahoma, sans-serif;
}
```

Comment (gergana95) [reply index.css]: done!

File: index.html (hunk 1 above)
Comment (gergana95) [reply index.html]: done!

File: index.css (hunk 2 above)
Comment (gergana95) [reply index.css]: FIXED!

File: index.css (hunk 3 above)
Comment (gergana95) [reply index.css]: fixed!

File: index.css (hunk 4 above)
Comment (gergana95) [reply index.css]: FIXED

File: index.css (hunk 6 above)
Comment (gergana95) [reply index.css]: added wrapper width and centralized it!

File: index.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+        url('fonts/subset-LeagueSpartan-Bold.woff') format('woff'),
+        url('fonts/subset-LeagueSpartan-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+/* Header styling */
+.wrapper {
+    width: 1630px;
+    margin: 12px auto;
```
Comment (superKalo) [inline index.css]: Set the 12px margin on the header instead, because it will cause you troubles afterwards (I'll explain why)

File: index.css [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+/* Header styling */
+.wrapper {
+    width: 1225px;
+    margin: 12px auto;
+}
+
+.header {
+    height: 44px;
```
Comment (superKalo) [inline index.css]: Use clearfix, no fancy height ;-)

File: index.html [hunk 11]
```diff
@@ -1,14 +1,27 @@
...
 <body>
+<div class="wrapper">
+     <header class="header">
+        <nav class="header__nav">
+            <img class="header__logo" src="images/logo.svg" alt="header logo" width="16px" height="44px" />
+            <ul class="list">
+                <li class="list__item"><a class="list__link font-primary" href="#">stories</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">events</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">places</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">boards</a></li>
```
Comment (superKalo) [inline index.html]: I think you need to use the `font-primary` class on the parent, only once, no need to use it on every single `<a>` tag.

File: index.html [hunk 12]
```diff
@@ -1,14 +1,27 @@
...
+    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Surfing</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="index.css" />
 </head>
 <body>
+<div class="wrapper">
+     <header class="header group">
+        <img class="header__logo left" src="images/logo.svg" alt="header logo" width="16px" height="44px" />
```
Comment (superKalo) [inline index.html]: `width="16px" height="44px"` are invalid values. [See here](https://www.w3schools.com/tags/att_img_width.asp).

**Comment by superKalo** [comment]
All right! You have my thumbs up now, 👍

**Pull request merged**